demos/projects/ST/b-l475e-iot01a/config/demo_config.h

```diff
@@ -507,10 +507,16 @@ static unsigned char root_cert_array[] = {
  */
 #define WIFI_SECURITY_TYPE  WIFI_ECN_WPA2_PSK
 
-#define democonfigADU_DEVICE_MANUFACTURER "ST"
-#define democonfigADU_DEVICE_MODEL        "STM32L475"
-#define democonfigADU_UPDATE_PROVIDER     "ST"
-#define democonfigADU_UPDATE_NAME         "STM32L475"
-#define democonfigADU_UPDATE_VERSION      "1.0"
+#ifdef democonfigENABLE_DPS_SAMPLE
+    #define democonfigCHUNK_DOWNLOAD_SIZE    256
+#else
+    #define democonfigCHUNK_DOWNLOAD_SIZE    1024
+#endif /* democonfigENABLE_DPS_SAMPLE */
+
+#define democonfigADU_DEVICE_MANUFACTURER    "ST"
+#define democonfigADU_DEVICE_MODEL           "STM32L475"
+#define democonfigADU_UPDATE_PROVIDER        "ST"
```

Comment:
just noting that the provider doesn't match the one in the script (I know you said you were going to update it separately.

Reply:
Already on my list!